app/Makefile

```diff
@@ -56,8 +56,7 @@ endif
 
 APP_LOAD_PARAMS = --curve secp256k1 $(COMMON_LOAD_PARAMS) --path $(APPPATH)
 
-NANOS_STACK_SIZE := 2050
-
+APP_STACK_MIN_SIZE := 3100
```

Comment:
Do we really want to pin the stack size? It could break the build if Ledger starts using just 1 more byte. I'd just stick to the default max stack size.

Reply:
We actually don't need it but it is a good way to explicitly check the stack size. Since the previous value was 2kB maybe we can set this to 2,5kB and not be on the edge